src/OpenTelemetry.Exporter.Zipkin/Implementation/TraceExporterHandler.cs

```diff
@@ -146,21 +146,44 @@ private string EncodeSpanId(ISpanId spanId)
 
         private ZipkinSpanKind ToSpanKind(ISpanData spanData)
         {
-            if (spanData.Kind == SpanKind.Server)
-            {
-                return ZipkinSpanKind.SERVER;
-            }
-            else if (spanData.Kind == SpanKind.Client)
+            var zipkinSpanKind = ZipkinSpanKind.CLIENT;
+
+            switch (spanData.Kind)
             {
-                if (spanData.HasRemoteParent.HasValue && spanData.HasRemoteParent.Value)
-                {
-                    return ZipkinSpanKind.SERVER;
-                }
+                case SpanKind.Server:
+                    zipkinSpanKind = ZipkinSpanKind.SERVER;
+                    break;
+                case SpanKind.Client:
+                    if (spanData.HasRemoteParent.HasValue && spanData.HasRemoteParent.Value)
+                    {
+                        zipkinSpanKind = ZipkinSpanKind.SERVER;
+                    }
+                    else
+                    {
+                        zipkinSpanKind = ZipkinSpanKind.CLIENT;
+                    }
+
+                    break;
+                case SpanKind.Producer:
+                    zipkinSpanKind = ZipkinSpanKind.PRODUCER;
+                    break;
+                case SpanKind.Consumer:
+
+                    if (spanData.HasRemoteParent.HasValue && spanData.HasRemoteParent.Value)
+                    {
+                        zipkinSpanKind = ZipkinSpanKind.PRODUCER;
```

Comment:
note: zipkin has different understanding of what is consumer span

https://github.com/apache/incubator-zipkin/issues/1931

It seems OT Consumer is Zipkin Server.